shared/ClrFacade/TestCases.cs

```diff
@@ -689,6 +689,35 @@ public static ITestGenericInterface<string>[] CreateTestArrayGenericInterface()
          result[i] = new ImplITestGenericInterface();
       return result;
    }
+   
+   public static object IsNull(object obj)
+   {
+      return (obj == null); 
+   }
+
+   public static object IsNA(object obj)
+   {
+      return obj;
+   }
+
+   public static object IsNaN(object obj)
+   {
+      if ((double?)obj == null) 
+         return false;
+
+      return double.IsNaN((double)obj);
+   }
+
+   public static object GetNull()
+   {
+      return null;
+   }
+
+   public static object GetNaN()
+   {
+      
+      return Double.NaN;
```

Comment:
In most cases we use the keyword instead of the base type. It's one of those conventions that we use in Suite.

eg. 
```
double.NaN 
```

You will see this all over the place in OSPSuite, but mainly with string manipulation
```
string.IsNullOrEmpty(someVariable)
```

rather than

```
String.IsNullOrEmpty(someVariable)
```